src/Protocol/Authenticator.h

```diff
@@ -3,7 +3,7 @@
 
 // Interfaces to the cAuthenticator class representing the thread that authenticates users against the official Mojang servers
 // Authentication prevents "hackers" from joining with an arbitrary username (possibly impersonating the server admins)
-// For more info, see http://wiki.vg/Session
+// For more info, see http://wiki.vg/Session -- page no longer exists
```

Comment:
similar to my first comment, the sessino page redirects to `Legacy_Mojang_Authentication`, but I think a link to `Protocol_Encryption#Authentication` mgiht be more suitable here.